platform/features/plot/src/elements/PlotTelemetryFormatter.js

```diff
@@ -63,6 +65,10 @@ define(
         };
 
         PlotTelemetryFormatter.prototype.formatRangeValue = function (value) {
+            if (typeof value === 'number') {
```

Comment:
What if value is a string representation of a number? Assuming string->number conversion has happened already by this point?